plugins/org.eclipse.n4js/src/org/eclipse/n4js/validation/N4JSJavaScriptVariantHelper.java

```diff
@@ -26,150 +27,237 @@
 	private final static String EXT_JSX = N4JSGlobals.JSX_FILE_EXTENSION;
 	private final static String EXT_N4JS = N4JSGlobals.N4JS_FILE_EXTENSION;
 	private final static String EXT_N4JSD = N4JSGlobals.N4JSD_FILE_EXTENSION;
+	private final static String EXT_N4IDL = N4IDLGlobals.N4IDL_FILE_EXTENSION;
+	private final static String EXT_IDL = N4IDLGlobals.IDL_FILE_EXTENSION;
```

Comment:
I'd remove all support for file extension ".idl" (but maybe discuss with Jens, first)